common/system.h

```diff
@@ -1295,6 +1298,21 @@ class OSystem : Common::NonCopyable {
 	 */
 	virtual Common::String getSystemLanguage() const;
 
+	/**
+	 * Suggest textures to render at the side of the game window.
+	 * This enables eg. Grim to render the game in a widescreen format.
+	 * 
+	 * The system must take ownership or the pointers will be freed
+	 * automatically.
+	 *
+	 * !!! ResidualVM specific method: !!!
+	 *
+	 * @param left			Texture to be used on the left
+	 * @param height		Texture to be used on the right
+	 */
+	virtual void suggestSideTextures(Common::ScopedPtr<Graphics::Surface>& left,
```

Comment:
After a more thorough look at the memory management here, I think it is incorrect because Graphics::Surface pointers need to have their 'free' method called to release their pixel data.
I suggest either:
* Common::SharedPtr along with Graphics::SharedPtrSurfaceDeleter is used instead of Common::ScopedPtr.
* Or preferably a plain Graphics::Surface pointer is passed as a parameter to suggestSideTextures, and is freed in GrimEngine::setSideTextures, explicitly stating in a Doxygen comment that the backend is expected to copy the surface data.